test/tasks/grunt-mocha.js

```diff
@@ -1,70 +0,0 @@
-'use strict';
-
-var fs = require('fs'),
-  path = require('path'),
-  shell = require('shelljs'),
-  kexec = require('kexec');
-
-module.exports = function (grunt) {
-  var file = grunt.option('file'),
-    debug = grunt.option('debug'),
-    inspector = grunt.option('inspector'),
-    fgrep = grunt.option('fgrep'),
-    helpers = ['test/support/modules_helper.js', 'test/support/test_helper.js', 'test/support/root_hooks.js'];
-
-  function getRelativePath(files) {
-    return files.map(function (helperPath) {
-      var fullPath = path.join(path.dirname(fs.realpathSync(__filename)), '../..', helperPath);
-      return path.relative(process.cwd(), fullPath);
-    });
-  }
-
-  function resolveTests(testString) {
-    return testString.split(',').map(function (test) {
-      var fullPath = path.join(process.cwd(), test);
-      return path.relative(process.cwd(), fullPath);
-    });
-  }
-
-  grunt.registerTask('mocha:webserver', 'Run the Mocha web server', function () {
-    kexec('test/web_server');
-  });
-
-  grunt.registerTask(
-    'mocha',
-    'Run the Mocha test suite.\nOptions:\n  --file=<file> e.g. --file=test/rest/auth.test.js\n  --debug will debug using standard node debugger\n  --inspector will start with node inspector',
-    function () {
-      var runTests = getRelativePath(helpers).concat(['test/realtime/*.test.js', 'test/rest/*.test.js']).join(' ');
```

Comment:
previously we were providing a list of test files explicitly here: `test/realtime/*.test.js`, `test/rest/*.test.js`.
now we don't and `mocha` loads `test/all.js` file which imports `rest` and `realtime` tests. I assume `all.js` wasn't used at all while we had `grunt mocha` task.

and then, it seems like we can remove `all.js` too. we already require `modules_helper` and `test_helper` modules via `.mocharc.js` config and we can add `spec` prop to that to specify `test/realtime/*.test.js` and `test/rest/*.test.js` test files.

@owenpearson WDYT?

Reply:
and for browsers it seems like everything is loaded using `test/support/browser_setup.js` script with data from `test/support/browser_file_list.js`, so it also doesn't use `all.js`.

Reply:
nice, yeah you were right that `all.js` wasn't being used anymore. and I've added a `spec` field to the mocha node config